pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -389,8 +418,7 @@ def default_compilation_pass(
         # https://cqcl.github.io/pytket-qiskit/api/index.html#default-compilation
         # Edit this docs source file -> pytket-qiskit/docs/intro.txt
         if optimisation_level == 0:
-            if self._standard_gateset:
```

Comment:
I removed this "if" statement as we need to apply the rebase regardless of whether the device supports CX or ECR.

Perhaps we should check that the circuit satisfies the `GateSetPredicate` here instead? In that case we wouldn't even need a rebase.